pkg/remote/github/init.go

```diff
@@ -50,8 +50,9 @@ func Init(version string, alerter *alerter.Alerter,
 	remoteLibrary.AddEnumerator(NewGithubRepositoryEnumerator(repository, factory, provider.Config))
 	remoteLibrary.AddDetailsFetcher(github.GithubRepositoryResourceType, common.NewGenericDetailsFetcher(github.GithubRepositoryResourceType, provider, deserializer))
 
-	supplierLibrary.AddSupplier(NewGithubMembershipSupplier(provider, repository, deserializer))
```

Comment:
The Membership supplier should not be removed here as we only handle team membership in this PR